README.md

```diff
@@ -135,6 +143,9 @@ KarhooApi.authService.login(token = "eyJra...zRQ7Tn3A").execute {
 }
 ```
 
+It is also possible to verify issued tokens
+by using [the token exchange endpoint directly from our API explorer](https://developer.karhoo.com/reference#post_oauth-v2-token-exchange).
```

Comment:
It's just once the client is registered on our side, but good thing mentioning it 👍